lib/Controller/AppointmentConfigController.php

```diff
@@ -42,26 +43,36 @@ class AppointmentConfigController extends Controller {
 	/** @var string|null */
 	private $userId;
 
+	/** @var LoggerInterface */
+	private $logger;
+
 	public function __construct(string $appName,
 								IRequest $request,
 								AppointmentConfigService $appointmentService,
+								LoggerInterface $logger,
 								?string $userId) {
 		parent::__construct($appName, $request);
 		$this->appointmentConfigService = $appointmentService;
 		$this->userId = $userId;
+		$this->logger = $logger;
 	}
 
 	/**
 	 * @NoAdminRequired
 	 */
 	public function index(): JsonResponse {
 		if ($this->userId === null) {
+			$this->logger->warning('User not found');
```

Comment:
Log `userId` as well. :)

Reply:
I can't if it's null :)

Reply:
🙈 I was probably looking somewhere else.